### PR TITLE
Section 2 - Will Littler

### PR DESCRIPTION
**Pull request by wlittler**
- wrote lesson zero test

### CODE REVIEW

File: test/01LessonOneTest.js [hunk 1]
```diff
@@ -5,36 +5,39 @@ var assert = require('assert');
...
-
+    return false;
 }
 
 function greaterThan(big, small) {
     // should return true if the first argument is greater than the second argument
     // and return false if the first argument is less than the second argument
-
+    return 4 > 3;
+    console.log(4 < 3);
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js]: @wlittler `big` and `small` are the values that are passed in to the function. You should be comparing `big` and `small`.

Like this:

```
 function greaterThan(big, small) {
      // should return true if the first argument is greater than the second argument
      // and return false if the first argument is less than the second 
      return big > small;
}
```

File: test/01LessonOneTest.js [hunk 2]
```diff
@@ -5,36 +5,39 @@ var assert = require('assert');
...
 }
 
 function lessThan(small, big) {
     // should return true if the first argument is less than the second argument
     // and return false if the first argument is greater than the second argument
+    return 3 < 4;
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js:27]: @wlittler Same comment as above.

File: test/01LessonOneTest.js [hunk 3]
```diff
@@ -5,36 +5,39 @@ var assert = require('assert');
...
 
 }
 
 function equalTo(a, b) {
     // should return true if the first argument is equal to the second argument
     // and should return true if the first argument is equal to the second argument
-
+    return 6 === 6;
+
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js]: Should be comparing `a` and `b` here, not `6 === 6`.

File: test/01LessonOneTest.js [hunk 4]
```diff
@@ -5,36 +5,39 @@ var assert = require('assert');
...
+    return 6 === 6;
+    
 }
 
 function notEqualTo(a, b) {
     // should return true if the first argument is not equal to the second argument
     // and should return false if the first argument is equal to the second argument
-
+    return 3 !== 4;
 }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js:41]: Should be comparing `a` and `b` here, not `3 !== 4`.

File: test/01LessonOneTest.js [hunk 5]
```diff
@@ -5,36 +5,39 @@ var assert = require('assert');
...
+    return a === b;
 
 }
 
 function notEqualTo(a, b) {
     // should return true if the first argument is not equal to the second argument
     // and should return false if the first argument is equal to the second argument
-
+    return a !== b;
 }
```
Comment (paulrenenichols) [inline test/01LessonOneTest.js:41]: Yay!, @wlittler!

**Comment by wlittler** [comment]
@paulrenenichols OK, that wasn't so bad. I'm sure you guys made this easy on purpose, really just copy paste and understand how to make the code go together without errors. Need to work on my syntax for commit messages - I like your "branch-serial

*description"

methodology, just need to clean my version up a bit.

File: test/02LessonTwoTest.js [hunk 6]
```diff
@@ -4,16 +4,31 @@ var assert = require('assert');
...
 function isEven(number) {
     // should return true is number is even (divisible by 2)
-
-}
+if (number % 2 === 0) {
+  return true }
+    else {
+      return false;
+    }
+};
```
Comment (paulrenenichols) [inline test/02LessonTwoTest.js:31]: We need to work on your formatting / indenting!

catbug

File: apps/02RockPaperScissors.js [hunk 7]
```diff
@@ -8,9 +8,52 @@ prompt.start();
...
+// if (hand1 !== 'rock','paper','scissors' || hand2 !== 'rock','paper','scissors') {
+//   return 'Does not compute! Use "rock", "paper", or "scissors" only.';
+// };
+
+// Valid input tester
+if (hand1 !== 'rock' && hand1 !== 'paper' && hand1 !== 'scissors') {
+  return 'Does not compute! Use "rock", "paper", or "scissors" only.';
+} else if (hand2 !== 'rock' && hand2 !== 'paper' && hand2 !== 'scissors') {
+  return 'Does not compute! Use "rock", "paper", or "scissors" only.';
+} else {
```
Comment (paulrenenichols) [inline apps/02RockPaperScissors.js]: `Does not compute`. Nice one!

File: apps/03TicTacToe.js [hunk 8]
```diff
@@ -23,22 +23,39 @@ function printBoard() {
...
 
 function checkForWin() {
     // Your code here
+    return (horizontalWin() || verticalWin() || diagonalWin());
 }
 
 function ticTacToe(row, column) {
     // Your code here
+    board[row][column] = playerTurn;
+    if checkForWin (true) {
```
Comment (paulrenenichols) [inline apps/03TicTacToe.js]: It looks like you have a syntax error here. Do you see it?

Comment (wlittler) [inline apps/03TicTacToe.js]: Should be if (checkForWin = true) ?

Comment (wlittler) [inline apps/03TicTacToe.js]: shoot, I mean
if (checkForWin === true)

File: apps/03TicTacToe.js [hunk 9]
```diff
@@ -23,22 +23,39 @@ function printBoard() {
...
 
 function ticTacToe(row, column) {
     // Your code here
+    board[row][column] = playerTurn;
+    if (checkForWin() === true) {
+      console.log('Player ' + playerTurn + ' Won!')
+    }
+    else
+    {
+    playerTurn = (playerTurn === 'X') ? 'O' : 'X';
```
Comment (paulrenenichols) [inline apps/03TicTacToe.js]: Indentation!

File: apps/03TicTacToe.js [hunk 10]
```diff
@@ -23,22 +23,39 @@ function printBoard() {
...
 
 function checkForWin() {
     // Your code here
+    return (horizontalWin() || verticalWin() || diagonalWin());
 }
 
 function ticTacToe(row, column) {
     // Your code here
+    board[row][column] = playerTurn;
+    if (checkForWin() === true) {
```
Comment (paulrenenichols) [inline apps/03TicTacToe.js]: Why not this?

``` javascript
if (checkForWin())
```

File: apps/04TowersOfHanoi.js [hunk 11]
```diff
@@ -16,24 +16,50 @@ function printStacks() {
...
 function towersOfHanoi(startStack, endStack) {
     // Your code here
-
+    // stacks.startStack
+    if (isLegal(startStack, endStack)) {
+        movePiece(startStack, endStack);
+    } else {
+      console.log("You can't do that, try again!");
+    }
+    checkForWin();
```
Comment (paulrenenichols) [inline apps/04TowersOfHanoi.js:62]: Yay!

catbug

File: apps/07jquery-tic-tac-toe/script.js [hunk 12]
```diff
@@ -2,4 +2,51 @@
...
+      if ($(this).text() === 'X' || $(this).text() === 'O') {
+        $('#announce-winner').text('Pick another square!')
+        }
+      else {
+      $(this).text(playerTurn);
+      checkForWin();
+      playerTurn === 'X' ? playerTurn = 'O' : playerTurn = 'X';
+      $('span.player').text(playerTurn);
+      }
+    });
```
Comment (paulrenenichols) [inline apps/07jquery-tic-tac-toe/script.js:18]: Yay, @wlittler!

File: apps/08jquery-towers-of-hanoi/style.css [hunk 13]
```diff
@@ -1,3 +1,8 @@
+.hanoiHolder {
+  transform: rotate(-90deg);
```
Comment (paulrenenichols) [inline apps/08jquery-towers-of-hanoi/style.css:2]: Great job! Something about your rotation is blocking the win message, though.

**Pull request closed without merging**